ext/intl/calendar/calendar_class.h

```diff
@@ -33,13 +33,23 @@ typedef struct {
 	intl_error  err;
 
 	// ICU calendar
+#ifndef __cplusplus
```

Comment:
I'm not so sure this trick is a good idea. Is there any guarantee in the C++ standard on the layout of a unique_ptr?
Perhaps we should convert the entire intl extension to C++ first before doing a change in a common file.